src/js/highscore.js

```diff
@@ -96,36 +199,59 @@ var highscore = (function () {
         })
     }
 
-    $('#highscoreOnUsername').submit(function () {
+    function onUserNameChange() {
+        const selectedUserName = $("#username").val();
         $("#row").empty();
-        highscoreOnName.length = 0;
+        if(selectedUserName === '') {
+            higscoreToHtml();
+            return;
+        }
+
+        hue = 0;
         $.each(highscoreObject, function (nb, obj) {
             if (nb === "scores") {
                 $.each(obj, function (nb, obj) {
                     makeHighscoreHeader(obj.DisplayName);
-                    var found = false;
+
+                    let highScoreInfo = { found: false };
                     $.each(obj.scores, function (nb, obj) {
-                        if (obj.playerName.toLowerCase() === $("#username").val().toLowerCase()) {
-                            highscoreOnName[0] = [obj.index, obj.playerName, obj.score];
-                            found = true;
-                        } else if (found == false) {
-                            highscoreOnName[0] = ["-", "-", "-"]
+                        if (obj.playerName.toLowerCase() === selectedUserName.toLowerCase()) {
+                            highScoreInfo = { ...obj, found: true};
                         }
                     })
-                    $.each(highscoreOnName, function (nb, obj) {
+
+                    if(highScoreInfo.found) {
                         $("tbody").last().append($("<tr>").addClass("usernameHighlight")
                             .append($("<td>").addClass("rank")
-                                .text(obj[0]))
+                                .text(highScoreInfo.index))
+                            .append($("<td>").addClass("avatar")
+                                .append($("<img>").attr("alt", "Avatar of " + highScoreInfo.playerName)
+                                    .attr("src", 'https://crafatar.com/avatars/' + uuidByPlayerName[highScoreInfo.playerName])))
```

Comment:
Please use the avatar with overlay - many skins rely on it.
See: https://github.com/NearVanilla/website/blob/master/src/js/members.js#L54
```suggestion
                                    .attr("src", 'https://crafatar.com/avatars/' + uuidByPlayerName[highScoreInfo.playerName] + '?overlay=true')))
```

Is it possible to skip adding the src when the result is `undefined`?